examples/star-wars/data/database.js

```diff
@@ -1,3 +1,4 @@
+"use strict";
```

Comment:
Can you throw these below the license header as in the rest of the files in `src/`